src/componentfuncs/Comments.jsx

```diff
@@ -19,4 +23,56 @@ export function RetrieveComments (props) {
   }
 }
   
+export function LeaveComment (props) {
+  const [comment, setComment] = useState()
+  const [validity, setValid] = useState(true)
+  const [inputFeedback, setInputFeedback] = useState('success')
+
+
+  const showing = props.show
+  const currentId = props.id
+
+  const commentValidate = (comment) => {
+    if (typeof comment === 'string' && comment.length > 15){
+      postComment(currentId, comment)
```

Comment:
What happens if the post fails? In order to complete this ticket, we need to handle the possibility of error upon posting